src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -715,6 +725,15 @@ internal virtual bool SupportsExtensionMethods
                     break;
                 }
 
+                // PROTOTYPE consider taking this out of the loop. tuples are always at the end of hierarchy
+                // PROTOTYPE match this in other lookups
+                var tuple = currentType as TupleTypeSymbol;
+                if (tuple != null)
+                {
+                    currentType = tuple.UnderlyingTupleType;
```

Comment:
I've added a "PROTOTYPE" since this is not in scope of this particular change. Will definitely need that when implementing symbols info.